src/games/strategy/triplea/ai/proAI/util/ProUtils.java

```diff
@@ -46,6 +47,26 @@ public ProUtils(final ProAI ai) {
     return unitTerritoryMap;
   }
 
+  public List<PlayerID> getEnemyPlayersInTurnOrder(final PlayerID player) {
+    final GameData data = ai.getGameData();
+    final List<PlayerID> enemyPlayers = new ArrayList<PlayerID>();
+    GameSequence sequence = data.getSequence();
```

Comment:
Good catch. I usually just leave on the auto-format setting to add final where it can be added. Looks like that setting was turned off during the changing of the formatting profile.